.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
```

Comment:
[nit] Not sure that we need node.js version matrix for browser tests if we don't have plan to launch it actually for several versions

Reply:
Done